tests/test_gosymbolextractor.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+import shutil
+import tarfile
+
+import pytest
+
+from system.plugins.gosymbolextractor.extractor import GoSymbolExtractor
+from system.helpers.artefact_schema_validator import ArtefactSchemaValidator
+
+
+TESTDATA_DIR = 'testdata'
+
+PATHS = [
```

Comment:
Would be great to put it into json file so the tests are independent of its input.